htmLawed.php

```diff
@@ -814,7 +814,7 @@ function hl_tag($t)
                 $v = str_replace('­', ' ', (false !== strpos($v, '&') ? str_replace(['&#xad;', '&#173;', '&shy;'], ' ', $v) : $v)); // double-quoted char: soft-hyphen; appears here as "­" or hyphen or something else depending on viewing software
                 if ('srcset' === $k) {
                     $v2 = '';
-                    $pattern = "/(?:[^\"'\s]+\s*(?:\d+w|\d+(?:\.\d+)?x)+)/";
+                    $pattern = "/(?:[^\"',\s]+(?:\s(?:\d+w|\d+(?:\.\d+)?x))?)/";
```

Comment:
Ugh, this really needs test coverage. If I am reading https://html.spec.whatwg.org/multipage/images.html#srcset-attribute right, it should be `\s+`.

Reply:
Hm indeed, I'll rework it and I'll try to add tests

Reply:
Fixed the regex in https://github.com/fossar/HTMLawed/pull/9/commits/51698deb86146d82816d13b7812b2cfad69c9f6e

Added tests for this case along Github Actions workflow in https://github.com/fossar/HTMLawed/pull/9/commits/e012d637ce982a481b8f5c714800e304a12b0c1e

Reply:
Poke @j0k3r